cdc/redo/manager.go

```diff
@@ -493,11 +493,10 @@ func (m *ManagerImpl) flushLog(
 }
 
 func (m *ManagerImpl) onResolvedTsMsg(span tablepb.Span, resolvedTs model.Ts) {
-	value, loaded := m.rtsMap.Load(span)
-	if !loaded {
-		panic("onResolvedTsMsg is called for an invalid table")
+	// It's possible that the table is removed while redo log is still in writing.
```

Comment:
Why does this happen？

Reply:
`redo_log_worker` is running background. `SinkManager.AsyncStopTable` won't wait all tasks get finished in that worker. So a table can be removed before `onResolvedTsMsg` is called.